modin/conftest.py

```diff
@@ -211,7 +211,7 @@ def __init__(self, modin_frame):
         self._modin_frame = modin_frame
 
     def finalize(self):
-        self._modin_frame.finalize()
+        self._modin_frame._finalize()
```

Comment:
I think this looks weird, we're using private API here now :)

Reply:
Maybe this should be public? I don't imagine it being used outside of this pass-through, but I agree it does look weird.

Reply:
Okay, let's keep it public but since we state the following `Direct partitions manipulation by this class is prohibited except cases if an operation is striclty private or protected and called inside of the class only` about `BasePandasFrame` in our documentation I suggest adding `finalize` method to the partition manager to perform `drain_call_queue()` on partitions.

Reply:
That seems fine @YarShev.

Reply:
Done!